### PR TITLE
Upgraded jcifs and JSch to their latest versions

### PR DESCRIPTION
**Pull request by sandamal**
Add jcifs 1.3.17 and JSch 0.1.51 to Orbit.
https://wso2.org/jira/browse/ESBJAVA-3449

### CODE REVIEW

File: jsch/0.1.51.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            com.jcraft.jsch.*;version="${version.jsch}"
+                        </Export-Package>
+                        <Import-Package>
+                            com.jcraft.jzlib;resolution:=optional,
```
Comment (manoj-kumara) [inline jsch/0.1.51.wso2v1/pom.xml]: Please add explicitly not to import packages which export by this bundle.

ex: 
!com.jcraft.jsch.*;

File: jcifs/1.3.17.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            jcifs.*;version="${version.jcifs}"
+                        </Export-Package>
+                        <Import-Package>
+                            javax.servlet;resolution:=optional,
```
Comment (manoj-kumara) [inline jcifs/1.3.17.wso2v1/pom.xml]: Please add explicitly not to import packages which export by this bundle.

File: jcifs/1.3.17.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>jcifs.wso2</groupId>
+    <artifactId>jcifs</artifactId>
+    <version>${orbit.version.jcifs}</version>
```
Comment (kasunbg) [inline jcifs/1.3.17.wso2v1/pom.xml]: The maven property substitution is not allowed in /project/parent/(groupId|artifactId|version) or in
/project/(groupId|artifactId|version). So, please hard-code the version here.

See here for more info - http://stackoverflow.com/a/3685248

File: pom.xml [hunk 4]
```diff
@@ -58,6 +58,8 @@
         <module>cassandra-thrift/1.2.13.wso2v4</module>
         <module>bsf/3.0.0.wso2v4</module>
         <module>lucene/4.10.3-wso2v1</module>
+        <module>jcifs/1.3.17.wso2v1</module>
+        <module>jsch/0.1.51.wso2v1</module>
     </modules>
```
Comment (kasunbg) [inline pom.xml]: Please remove these. Orbit parent pom is not used anymore.

**Pull request merged**